### PR TITLE
Feature/requests review page

### PR DESCRIPTION
**Pull request by LiveBacteria**
Fully implemented FE capability to view and manipulate requests. 

Updated FS logic to allow for the above.

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -153,7 +153,16 @@ module.exports = {
...
         const reviews = await Review.find({ userId: userId });
-        return res.status(201).json({ reviews });
+        const requests = await Request.find({
+          selectedReviewer: userId,
+        });
+        const embeddedReviewArray = [];
+
+        for (let i = 0; i < requests.length; i++) {
+          embeddedReviewArray.push(requests[i].embeddedReview);
+        }
```
Comment (Affeeq) [inline server/controllers/user.js:163]: you might not need to create a variable for an array as **find** should return an array. you can use **select** to only extract the embeddedReview

File: client/src/pages/Reviews.js [hunk 2]
```diff
@@ -95,10 +113,55 @@ const Reviews = () => {
...
+    }
+    setSelectedTab(updatedTab);
+  };
+
   const handleListItemClick = (event, index, props) => {
     setSelectedIndex(index);
   };
 
+  const renderItems = (target) => {
+    console.log(state);
```
Comment (Affeeq) [inline client/src/pages/Reviews.js:135]: remove this

File: server/controllers/user.js (hunk 1 above)
Comment (LiveBacteria) [reply server/controllers/user.js:163]: Interestingly, I tried .select("embeddedReview"), but ran into issues where the request was returned null.

I will retry that once more before merging.

File: client/src/pages/Reviews.js (hunk 2 above)
Comment (LiveBacteria) [reply client/src/pages/Reviews.js:135]: The console.log()? Yes, I missed that on my sweep through tidying up.

File: server/controllers/user.js (hunk 1 above)
Comment (LiveBacteria) [reply server/controllers/user.js:163]: Ok, I tried using .select(), it doesn't return what I am trying to work with.

**Pull request merged**